src/locales/de/common.json

```diff
@@ -88,6 +88,39 @@
       "vars": "Variablen"
     },
     "search": "Suche",
+    "searchForm": {
```

Comment:
We recommend that each PR only have one change. Can these i18n changes be isolated into a separate PR?